api/go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/regen-network/regen-ledger/api
+module github.com/regen-network/regen-ledger/api/v2
```

Comment:
Why was this done? This seems like a spurious go module major version bump. Were there actually breaking changes? I thought we weren't releasing breaking proto changes to production. With proper buf breaking checks, the api go.mod should never have been bumped past v1.